Detectors/MUON/MCH/Workflow/include/MCHWorkflow/DataDecoder.h

```diff
@@ -0,0 +1,132 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    DatDecoder.cxx
+/// \author  Andrea Ferrero
+///
+/// \brief Implementation of a data processor to run the raw decoding
+///
+
+#include <random>
```

Comment:
Does the DataDecoder.h needs to be public (i.e. in include/MCHWorkflow) ? 
If it does (and even if it does not, by the way), you should limit as much as possible the number of includes it uses. Do you need as many ? Move as much as possible in the .cxx.

Reply:
I guess It needs if we want to use the decoder independently of the DPL, for example when processing the raw data directly in the QC.

I have removed all the unnecessary headers.